comicrelief/campaign
# Feature/plat 199 yamlform queue

Fixes https://jira.comicrelief.com/browse/PLAT-199
## Changes proposed in this pull request
- [ ] Implement YAML Form and custom queue handler (from https://github.com/pvhee/yamlform_queue)
## How to test
- Install `yamlform`,  `yamlform_queue`, `rabbitmq` and `rabbitmq_example`
- Create a new `yamlform`
- Edit form and within `Email / Handlers` add `Queue`
- Add `$settings['queue_service_queue1'] = 'queue.rabbitmq';` to `settings.local.php`
- Make a test submission, this should end up in the queue
